training/distributed_training/pytorch/model_parallel/vit/vit_hooks.py

```diff
@@ -0,0 +1,80 @@
+# Third Party
```

Comment:
This should be in smp?

Reply:
Since those changes are not released in DLC yet, I have added here.